Resources/Prototypes/_Nyano/Entities/Objects/Specific/Mail/base_mail.yml

```diff
@@ -96,8 +95,17 @@
   - type: DamageOtherOnHit
     damage:
       types:
-        Blunt: 5
+        Blunt: 1 # Frontier - 5<1, death by 1000 cuts.
   - type: CargoSellBlacklist
+  - type: Food # Frontier - Moth food
+    requiresSpecialDigestion: true
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 1
+        reagents:
+        - ReagentId: Fiber
+          Quantity: 1
```

Comment:
Moths absolutely should not have a free unlimited food source at will. Please remove this.

Reply:
Changed it to nothing to keep it as RP only, cannot be used as real food now.

Reply:
the opened mail can still be food i guess? wait no.  If the mail package entity gets deleted when you destroy mail, then we can make opened mail edible, otherwise yea cant have it sadly